### PR TITLE
[OYPD-281] Add some Simple Content to Camp page. Add blog posts related to Camp pages.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-281?focusedCommentId=82500&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-82500
D.o https://www.drupal.org/node/2858925

Steps to review:
- [x] Visit demo camps /camps/camp-colman, /camps/camp-orkila, & /camps/camp-terry
- [x] See Simple Content on pages
- [x] See blog posts on pages, below simple content

## Two tests fail.
1. Feature: Latest Blogs Paragraphs "Scenario: Paste latest blog post", fails on `And I should see "Mango-Avocado Salsa"` with the message `(The text "Mango-Avocado Salsa" was not found anywhere in the text of the current page.)`. This is demo content and should probably not be relied on for tests.
image

2. UNRELATED Feature: Static Paragraphs "Scenario: Create Small Banner", fails on `When I press "Add Small Banner" in the "header_area" ` with the message `(The button 'Add Small Banner' was not found in the region 'header_area' on the page http://openy-dev.ffwua.com/build709/node/add/landing_page)`. This test failure was missed on https://github.com/ymcatwincities/openy/pull/215 for https://propeople-us.atlassian.net/browse/OYPD-293 which removed the small banner from the header content field options.
image

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build709 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build709 |
| Installation process | http://install.openy-dev.ffwua.com/build709/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/709/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build709/profiles/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build710 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build710 |
| Installation process | http://install.openy-dev.ffwua.com/build710/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/710/

File: modules/openy_features/openy_demo_content/modules/openy_demo_ncamp/config/install/migrate_plus.migration.node_blog_camp.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+      bg_color: 8
+      color: 1
+      category: 2
+      related: 1
+      promote: 1
+      camp: 3
+      body: >
+        <p><span>Located on the Raging River in&nbsp;Preston, Washington, YMCA Camp Terry provides outdoor fun for all. Originally deeded to the YMCA by Clarence C. Terry&nbsp;in 1942 for $10, YMCA Camp Terry has grown&nbsp;to provide&nbsp;a range of programs including Outdoor Day Camp,&nbsp;Family Camp Overnights and Outdoor Environmental Education. Operated by the Sammamish Family YMCA, YMCA Camp Terry most recently&nbsp;added a climbing wall and 250-foot zip line to its facilities producing more summer fun than ever!</span><br>
+        YMCA Outdoor Day Camp</p>
+        <ul class="ms-rteStyle-Normal"><li>
```
Comment (alexschedrov) [inline modules/openy_features/openy_demo_content/modules/openy_demo_ncamp/config/install/migrate_plus.migration.node_blog_camp.yml]: Let's remove class.

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build710/profiles/openy/build/reports/behat

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by alexschedrov** [comment]
For some reason some blog posts no longer have images
image

**Comment by Kerby70** [comment]
Those are new blog posts, content from random pages on seattleymca.org, I did not realize the importance of the style photo card.

**Comment by alexschedrov** [comment]
@Kerby70 let's just change style to some that doesn't require image.

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/716/

**Build Log**
*last 25 lines*

 ```
[...truncated 281 lines...]
stdout: You are about to create a /var/www/build716/sites/default/settings.php file and DROP all tables in your 'drupal716' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=6    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by Kerby70** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/718/

**Build Log**
*last 25 lines*

 ```
[...truncated 272 lines...]
stdout: You are about to create a /var/www/build718/sites/default/settings.php file and DROP all tables in your 'drupal718' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=5    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by Kerby70** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/721/

**Build Log**
*last 25 lines*

 ```
[...truncated 281 lines...]
stdout: You are about to create a /var/www/build721/sites/default/settings.php file and DROP all tables in your 'drupal721' database. Do you want to continue? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=14   changed=6    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build722 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build722 |
| Installation process | http://install.openy-dev.ffwua.com/build722/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/722/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build722/profiles/openy/build/reports/behat

**Comment by alexschedrov** [comment]
@Kerby70 
1. Let's fix tests for blog posts. There's another task to move away from using demo content in behat tests.

2. Test for paragraph should be fixed in terms of https://propeople-us.atlassian.net/browse/OYPD-334

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build725 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build725 |
| Installation process | http://install.openy-dev.ffwua.com/build725/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/725/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build725/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**